health.js

```diff
@@ -0,0 +1,180 @@
+import React, { useContext, useReducer, useEffect, useState } from 'react';
```

Comment:
What is this new file? We already have an example app available for testing/development, please use it. A more friendly example app would be a very welcome contribution, but please open another PR for that